src/main/java/com/obj/nc/security/config/JwtSecurityConfig.java

```diff
@@ -55,9 +57,17 @@ public AuthenticationManager authenticationManagerBean() throws Exception {
 	
 	@Override
 	protected void configure(HttpSecurity httpSecurity) throws Exception {
-		httpSecurity.csrf().disable()
+		httpSecurity
+				.cors().configurationSource(request -> {
+					CorsConfiguration cors = new CorsConfiguration();
+					cors.setAllowedOrigins(Arrays.asList("http://localhost:9000"));

```

Comment:
toto sa mi nejak nezada,.. ten localhost aj s portom,.. 
nemalo by to byt v nejakej configuraci?

Reply:
neviem ako sa toto zvykne robit,
toto som pridal, aby Adrian mohol vyvijat FE voci NC nasadenemu na gcp, od NC dostaval taketo odpovede:

![image](https://github.com/com-obj/notiflow/assets/89594571/4270c850-a58b-40ff-a5fe-e0f2a6df0980)

pls aku konfiguraciu mas na mysli? spravit tomu config property napr. "cors.allowedOrigins" ?

Reply:
hmm,.. zvlastne,.. nie uplne rozumiem preco to nastava ale tak ok,..
ano config property cors.allowedOrigins ,.. aby sme to aj nemali niekde vhodene vnutri kodu

Reply:
done: https://github.com/com-obj/notiflow/pull/36